docs/pages/includes/database-access/aws-create-db-configuration-dynamic-resources.mdx

```diff
@@ -0,0 +1,30 @@
+Databases can be registered dynamically by Discovery Service, `tctl`, etc.
+
+Generate a Database Service configuration that monitors the dynamic database
+resources:
+
+<Tabs>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
```

Comment:
The difference between tab items is minimal here, so I would recommend using a `Var` or the Cloud URL for both. (Same with the other partial that includes `Tabs` like this)